proxy/api/src/http/project.rs

```diff
@@ -507,7 +507,7 @@ mod test {
             },
             "shareableEntityIdentifier": format!("%{}", meta.id.to_string()),
             "stats": {
-                "branches": 1,
+                "branches": 2,
```

Comment:
Is this enough of a test?

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1278/commits/92d6601c9e38f3a7efe28ad6832a7e03f1244a5b